app/assets/stylesheets/application.css.scss

```diff
@@ -40,6 +40,27 @@ legend {
   };
 }
 
+.reminder-container {
+  text-align: center;
+  margin: auto;
+  width: 80%;
+  border: 2px $dirty-white solid;
+  border-radius: 50px;
+  background-color: rgba(0,0,0,0.15);
+
+  h2 {
+    text-shadow: 0px 3px 0px #ffffff,
+      0px 14px 10px rgba(0,0,0,0.15),
+      0px 24px 2px rgba(0,0,0,0.1),
+      0px 34px 30px rgba(0,0,0,0.1);
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`